superset/common/query_context.py

```diff
@@ -265,7 +265,7 @@ def get_df_payload(  # pylint: disable=too-many-locals,too-many-statements
             except QueryObjectValidationError as ex:
                 error_message = str(ex)
                 status = utils.QueryStatus.FAILED
-            except Exception as ex:  # pylint: disable=broad-except
+            except KeyError as ex:
```

Comment:
There is a lot that can go wrong here, as the `try` wraps the `get_query_result` method - I'd recommend leaving this one at the broad exception. Errors are intentionally swallowed and reported for this chunk of code.

Reply:
Sure, no problem - I'll bring it back.